tests/plugins/test_add_filesystem.py

```diff
@@ -213,6 +223,11 @@ def test_add_filesystem_plugin_generated(tmpdir, docker_tasker, scratch):
     mock_koji_session(scratch=scratch)
     mock_image_build_file(str(tmpdir))
 
+    if reactor_config_map:
+        workflow.plugin_workspace[ReactorConfigPlugin.key] = {}
+        workflow.plugin_workspace[ReactorConfigPlugin.key][WORKSPACE_CONF_KEY] =\
+            mocked_reactorconfig({'version': 1, 'koji': {'hub_url': KOJI_HUB,
+                                                         'root_url': '', 'auth': {}}})
```

Comment:
move this code and all similarly instances to a common function that takes workflow and reactor_config_map as arguments and sets the workflow configuration appropriately.

Reply:
ok